default.py

```diff
@@ -111,6 +110,11 @@ def clean(self, video_type):
         :rtype: (list, int, int)
         :return: A list of the filenames that were cleaned, as well as the number of files cleaned and the return status.
         """
+        expired_videos=kwargs.get('expired_videos', None)
+        if expired_videos!=None:
+            ImmediateFileClean = True
+        else:
+            ImmediateFileClean = False
```

Comment:
Please do not use CamelCase for variable names. I use ``lowercase_with_underscores`` throughout the code.